src/ovms.h

```diff
@@ -80,6 +80,9 @@ typedef enum OVMS_BufferType_enum {
     OVMS_BUFFERTYPE_CPU,
     OVMS_BUFFERTYPE_CPU_PINNED,
     OVMS_BUFFERTYPE_GPU,
+    OVMS_BUFFERTYPE_OPENCL,
```

Comment:
Maybe we should somehow comment those types above which are unused? CPU_PINNED/GPU might be misleading, in fact those are not meant to be used

Reply:
The same for HDDL, hddl is discontinued afaik